infra/Pos/Communication/Relay/Logic.hs

```diff
@@ -314,16 +315,18 @@ addToRelayQueue inv = do
     else
         atomically $ writeTBQueue queue (SomeInvMsg inv)
 
-relayWorkers :: forall m .
-             ( Mockable Throw m
-             , RelayWorkMode m
-             , MonadMask m
-             , MonadReportingMem m
-             )
-             => m (Set NodeId) -> OutSpecs -> ([WorkerSpec m], OutSpecs)
-relayWorkers getPeers allOutSpecs =
+relayWorkers
+    :: forall m.
+       ( Mockable Throw m
```

Comment:
Actually it's default/hindent styling. `::` and following `=>` and `->`s are indented to 4 spaces what's important.